### ISSUE TITLE
Support for `insteadOf` in `Gina browse` command

### ISSUE THREAD

**Issue by jearbear** [lambdalisue/gina.vim#249]
I have this in my `.gitconfig`:
```
[url "git@github.com:"]
    insteadOf = gh:
```

The current method of fetching the remote url seems to rely on `git config` and effectively does `git config remote.origin.url`, but this will provide the url with `gh:...` which doesn't work with `Gina browse`, giving me this error:
```
[gina] No url translation pattern for "" is found.
```

I see two ways of fixing this:
1. Modify the [`get_remote_url` function](https://github.com/lambdalisue/gina.vim/blob/43b3be43d02e6aedb1cd839cf1ecfb61b0fd6981/autoload/gina/command/browse.vim#L133-L145) to make a call out to `git remote get-url origin` which will return the fully resolved path. A similar approach was done for vim-fugitive: https://github.com/tpope/vim-fugitive/pull/874, though I'm not sure how involved it'll be for gina.vim.
2. To avoid a call to `git`, perform some internal translation based on if `insteadOf` is present in the config. I imagine this is way more work than it's worth.

**Comment by jearbear** [comment]
Oh I realized the first solution isn't too hard. Does something like this seem reasonable?
```diff
diff --git a/autoload/gina/command/browse.vim b/autoload/gina/command/browse.vim
index 78e9de8..d713922 100644
--- a/autoload/gina/command/browse.vim
+++ b/autoload/gina/command/browse.vim
@@ -141,7 +141,11 @@ function! s:get_remote_url(git, commit1, commit2) abort
     endif
   endfor
   let remote_name = empty(remote_name) ? 'origin' : remote_name
-  return get(config, printf('remote.%s.url', remote_name), '')
+  let result = gina#process#call(a:git, ['remote', 'get-url', remote_name])
+  if result.status
+    throw gina#process#errormsg(result)
+  endif
+  return result.content[0]
 endfunction
 function! s:build_base_url(remote_url, scheme) abort
```

**Comment by lambdalisue** [comment]
It seems reasonable. Please send a PR 👍

**Comment by lambdalisue** [comment]
Closed by #250
